code/script.js

```diff
@@ -30,18 +33,141 @@ const showMessage = (message, sender) => {
+    <option value="Iced Americano">Iced Americano</option>
+    <option value="Iced Espresso">Iced Espresso</option>
+    <option value="Iced Caffé Latte">Iced Caffé Latte</option>
+  </select>
+`
+
+    // User makes selection. ('select') refers to id slect. The list above.
+  }
+  const coffeeKindSelect = document.getElementById('select');
+  coffeeKindSelect.addEventListener('change', () => {   //EventListener added to drop-down menu. Reacts when the drop-down option changes.
+    showMessage(coffeeKindSelect.value, 'user'); //User answer the selected coffeeKind
+
+    setTimeout(() => handleCoffeeOptionInput(coffeeKindSelect.value), 1000) //setTimeout -arrow- Passing function (not invoke)
+
+  });
 }
 
-// Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
+// Bot answer based up on selection. I added template literal: `${coffeeOption} so the bot begin answer with the option that user choosed.
+// Added 3 buttons of size alternatives (105,106,107). Same code as for hot and cold button in the start.
+const handleCoffeeOptionInput = (coffeeOption) => {
+  option = coffeeOption;
+  console.log(coffeeOption);
+  showMessage(`${coffeeOption}, excellent choice! What size do you want?`, 'bot');
+  inputWrapper.innerHTML = `
+    <button id="smallBtn">Small</button>
+    <button id="mediumBtn">Medium</button>
+    <button id="largeBtn">Large</button>
+  `
+  //When user click on size button it triggers answer from user to bot: small, medium or large
+  document.getElementById('smallBtn').addEventListener('click', () => {
+    showMessage('Small', 'user'); //User answer "small"
+    setTimeout(() => handleCoffeeSizeInput('Small'), 1000);
+  });
+
+  document.getElementById('mediumBtn').addEventListener('click', () => {
+    showMessage('Medium', 'user'); //User answer "medium"
+    setTimeout(() => handleCoffeeSizeInput('Medium'), 1000);
+  });
+
+  document.getElementById('largeBtn').addEventListener('click', () => {
+    showMessage('Large', 'user'); //User answer "large"
+    setTimeout(() => handleCoffeeSizeInput('Large'), 1000);
+  });
+}
+
+//Bot answer based up on selection. I added two template literals: ${coffeeSie} and ${option}
+const handleCoffeeSizeInput = (coffeeSize) => {
+  showMessage(`A ${coffeeSize} ${option} coming up. Are you satisfied with your order?`, 'bot');
+  inputWrapper.innerHTML = //innerHTML input field.
+    `<input id="confirm-input" type="text" />
+  <button class="send-btn" id="send" type="submit">
+    Send
+  </button>`
+
+  //User gets input field to write answer after choosing size.
+  const inPutField = document.getElementById('confirm-input');
```

Comment:
not sure if there is a reason for the P to be uppercase for the inPutField here.. otherwise it should be lower case.

Reply:
Thx!